portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Displays only courses that are part of a specified class.
+ */
+function filterCourseDisplay(courseType = 'course') {
+
+  // Clear all courses from the display.
```

Comment:
Should I put lines 36-39 in an if (courseType != 'course) block or something? Since in this case all courses will be displayed so there is no reason to remove any from the display

Reply:
Alternatively, when we loop through every course we can set the display to none if allCourses[i] is equal to courseType, otherwise set the display to visible. That way we can just have one loop.